tools/build-examples-sequential.sh

```diff
@@ -9,7 +9,7 @@ export CI='1'
 
 BUILD_TARGET=$1
 
-npx lerna run --stream --concurrency 1 --no-bail --scope @examples/${BUILD_TARGET}* reinstall
-npx lerna run --stream --concurrency 1 --no-bail --scope @examples/${BUILD_TARGET}* build
-npx lerna run --stream --concurrency 1 --no-bail --scope @examples/${BUILD_TARGET}* synth
+npx lerna run --stream --concurrency 1 --no-bail --scope ${BUILD_TARGET}* reinstall
+npx lerna run --stream --concurrency 1 --no-bail --scope ${BUILD_TARGET}* build
+npx lerna run --stream --concurrency 1 --no-bail --scope ${BUILD_TARGET}* synth
 
```

Comment:
If we now target only one test in the `BUILD_TARGET` we can remove this script, can't we? Or does you change only apply to PR builds and we still need this elsewhere (e.g. when running builds on `main`)?

Reply:
Yep, cleaned it up a bit